forum/settings.py

```diff
@@ -37,6 +37,15 @@
     config("ALLOWED_ENV_HOST"),
 ]
 
+if DEBUG:
+    import socket
+
+    hostname, _, ips = socket.gethostbyname_ex(socket.gethostname())
+    INTERNAL_IPS = [ip[: ip.rfind(".")] + ".1" for ip in ips] + [
+        "127.0.0.1",
+        "10.0.2.2",
+    ]
```

Comment:
It looks too complicated and doesn't feel right. You have a 'magic string' here (`10.0.2.2`), plus the rest of the code is not the easiest to read/understand.

Overriding `SHOW_TOOLBAR_CALLBACK` looks like more explicit option. Though, it's a good idea to make it conditional (allowed only when `DEBUG = True` and probably taking some other measures to ensure, that you'll never get DDT turned on at Production environment).

https://django-debug-toolbar.readthedocs.io/en/latest/configuration.html#show-toolbar-callback — documentation on the callback options.

At one of my projects, the corresponding part of `settings.py` looks like this:
```python
def show_toolbar(request):
    return not IS_PRODUCTION_ENV and DEBUG and USE_DEBUG_TOOLBAR and not running_tests()


DEBUG_TOOLBAR_CONFIG = {
    'SHOW_TOOLBAR_CALLBACK': show_toolbar,
}
```